### PR TITLE
graphQL Apollo Setup

### PR DESCRIPTION
**Pull request by NimSum**
# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

```Integration of graphQL
This setup allows us to create different types of queries and use them to create methods for our backend interaction!
If you have any improvements you would like to add, let me know!
Please refer to the comments I added for better clarity.
```

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling change (non-breaking change which adds styling)
- [ ] Other, Please explain:

## Has This Been Tested?
- [x] No
- [ ] Yes

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration:

```I do not know the Apollo query testing works yet, I'll look into it more```

# Checklist:
- [x] My code has no unused/commented out code
- [x] My code has no console logs
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

# Message for reviewer:

```I added comments to the parts that you may have questions for, ask me in person if you have more questions!```

### CODE REVIEW

File: src/utils/queries/clientQuery.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import ApolloClient from 'apollo-boost';
+
+const client = new ApolloClient({
+  uri: 'https://cors-anywhere.herokuapp.com/https://pure-castle-14648.herokuapp.com/graphql'
+})
+
+// This is the function that "posts" the query, the client automatically sends the query object from the function argument.
+// A failed query also triggers the catch method
```
Comment (ehk9000) [inline src/utils/queries/clientQuery.js]: Cool! Can we walk through the client together this afternoon? Is this where we would make all of our custom queries?

File: src/utils/queries/queryTypes.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import { gql } from 'apollo-boost';
+
+// This is how to create graphQL queries(ensure new line after template literal start)
+// The naming convention I saw on the docs looks like redux actions so thats how the queries are named
+// If you want to make a new query just follow the format below
```
Comment (ehk9000) [inline src/utils/queries/queryTypes.js]: Awesome! How will this play out with thunks?

**Comment by tiffanybacher** [comment]
Thank you for doing the heavy lifting and figuring out how to get this all set up! Looking forward to implementing it myself.

File: src/utils/queries/clientQuery.js (hunk 1 above)
Comment (tiffanybacher) [reply src/utils/queries/clientQuery.js]: Upvote!

**Pull request merged**